limiter.go

```diff
@@ -89,5 +89,5 @@ func (c *ConcurrencyLimiter) Wait() {
 
 // GetNumInProgress returns a (racy) counter of how many go routines are active right now
```

Comment:
it's racy indeed, as documented by the comment
(this should not be used for anything other than debugging)